File: src/android/com/kuackmedia/androidauto/models/Tag.kt
package com.kuackmedia.androidauto.models

import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class Tag(
  override val id: String,
  override val itemType: String,
  override val itemStyle: String = "grid",
  override val score: Double? = null,
  val name: String,
  val description: String,
  val isGenre: Boolean,
  val isStation: Boolean,
  val images: List<CoverImage>,
  val updateDate: Long,
  val imageUpdateDate: Long,
  val amount: Any?,
  val imageColorInfo: ImageColorInfo? = null
) : MediaItem()

File: src/ios/models /Tag.swift
import Foundation

struct Tag: MediaItem {
    let id: String
    let itemType: String
    let itemStyle: String
    let name: String
    let description: String
    let isGenre: Bool
    let isStation: Bool
    let images: [CoverImage]
    let updateDate: Int64
    let imageUpdateDate: Int64
    let amount: AnyCodable?
    let imageColorInfo: ImageColorInfo?
}
